.eslintrc.js

```diff
@@ -6,5 +6,7 @@ module.exports = {
   },
   rules: {
     "unicorn/prefer-module": "off",
+    "unicorn/prefer-spread": "off",
+    "unicorn/prefer-number-properties": "off",
```

Comment:
You can remove these two as they are redundant.

Reply:
I will take care of it, as the content of this PR is fine and then you can focus on other tasks

Reply:
thanks <3